johnmusiu/Maintenance-Tracker
# [chore 158382565] Implement LFA feedback

### What does this PR do?
Implements LFA feedback.

### Description of Task to be completed?
Implement proper class structure for user i.e save class variables in `self.variable` using the constructor.
Also updates readme.

### How should this be manually tested?
Checkout the changes made to the `models.py` file on this PR.
Also check the folder structure.

### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/158382565
